sdk/python/test/e2e/test_e2e_mpijob.py

```diff
@@ -52,19 +52,25 @@ def test_sdk_e2e_with_gang_scheduling(job_namespace):
     launcher = V1ReplicaSpec(
         replicas=1,
         restart_policy="Never",
-        template=V1PodTemplateSpec(spec=V1PodSpec(
-            containers=[launcher_container],
-            scheduler_name=get_pod_spec_scheduler_name(GANG_SCHEDULER_NAME),
-        )),
+        template=V1PodTemplateSpec(
+            metadata=V1ObjectMeta(annotations={'sidecar.istio.io/inject': "false"}),
```

Comment:
we can move but with this approach, we might end up having a lot of constants. Also, since `sidecar.istio.io/inject` is a standard annotation, this looks more readable to me. 

And it's consistent with what we already have in training utils. 
https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/utils/utils.py#L325

Any specific reason to move it?

Reply:
> Also, since sidecar.istio.io/inject is a standard annotation, this looks more readable to me.

We should avoid redeclaring the same words in many places since that causes bugs.

Reply:
> And it's consistent with what we already have in training utils.
https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/utils/utils.py#L325

It would be better to move `sidecar.istio.io/inject` to [sdk/python/kubeflow/training/constants/constants.py](https://github.com/kubeflow/training-operator/blob/d52e2b05143c97c79874aa71ff080dff665a90c7/sdk/python/kubeflow/training/constants/constants.py).

Reply:
okay, let me move it.